test/fixtures/v5/continue/sharp_left_default.json

```diff
@@ -12,10 +12,13 @@
         "es": "Continua cerrada a la izquierda",
         "fr": "Continuer franchement à gauche",
         "id": "Terus tajam kiri",
+        "it": "Continua Leggera svolta a sinistra",
```

Comment:
I think the terms "sharp" and "slight" could ignore in Italian, translating "sharp right" into only "right". Is this a good solution? Otherwise there would be many problems..